docs/timeseries.md

```diff
@@ -443,3 +443,47 @@ Example CSV response:
 | 415     | 'Unsupported Media Type': The user send an invalid `Accept` header.                                                                                                         |
 
 *The 400 code is also currently returned for queries where no time series data could be located for the request specified feature ID. The message returned with the response indicates this and it can be helpful to adjust the date ranges you are searching.
+
+## API Keys
+
```

Comment:
Can we put something in here like

> [!IMPORTANT]
> API keys not yet implemented but coming soon. More details to follow

To give us more time to work the details of the trusted partner key

Reply:
Updated the documentation to include a notice that we have not yet implemented API keys.